app/src/sdk/BffApiClient.test.js

```diff
@@ -7,7 +7,7 @@ describe("BffApiClient", () => {
     body: {},
     json: function () {
       return this.body;
-    }
+    },
```

Comment:
I think we decided to turn off trailing commas in prettier?

Reply:
Yes, we need to run the whole repo through a linting tool once this is merged!